Cargo.toml

```diff
@@ -17,9 +17,9 @@ readme = "README.md"
 
 [features]
 nightly = []
-serde_impl = ["serde", "serde_json"]
+serde_impl = ["serde", "serde_test"]
 
 [dependencies]
 clippy = { version = "0.*", optional = true }
 serde = { version = "^0.7", optional = true }
-serde_json = { version = "^0.7", optional = true }
+serde_test = { version = "^0.7", optional = true }
```

Comment:
[Cargo does not support optional dev-dependencies](https://github.com/rust-lang/cargo/issues/1596). So I can make it a dev dependency but it would mean that serde and serde_test are compiled even when running `cargo test` without `--features serde_impl`. We had decided on this approach in #48.